src/index.html

```diff
@@ -250,29 +250,29 @@ <h2 class="section-title">Contacts us</h2>
                   >
                 <textarea
                   class="info__input info__input-textarea"
-                  placeholder="Message"
-                >
-                </textarea>
-
-                <a
-                  class="info__button"
-                  href="#"
+                  placeholder="Message"></textarea>
+                <input
+                type="reset"
+                class="info__button"
+                value= "Send"
                 >
-                Explore
-                </a>
-              </div>
+              </form>
 
               <div class="info__text">
                 <p class="info__title">Phone</p>
-                <p class="info__value">+1 234 5555-55-55</p>
+                <a href="tel:+1 234 5555-55-55" class="info__value">
+                  +1 234 5555-55-55
+                </a>
                 <p class="info__title">Email</p>
-                <p class="info__value">hello@miami.com</p>
+                <a href="mailto:hello@miami.com" class="info__value">
+                  hello@miami.com
+                </a>
                 <p class="info__title">Address</p>
-                <p class="info__value">
+                <a href="https://maps.app.goo.gl/WFYCfC8zFWYR9Zmx7" class="info__value">
```

Comment:
all external links should be open in the new tab (again - checklist will help you)